package.json

```diff
@@ -4,19 +4,19 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn run build:api && yarn workspace @oasis/web run build",
+    "build": "yarn workspace @oasis/web run build",
     "build:api": "yarn workspace @oasis/api run build && yarn workspace @oasis/client-gql run build",
     "clean": "yarn workspaces foreach --include @oasis/api,@oasis/client-gql,@oasis/web -p run clean",
     "commit": "git add . && cz && git push",
-    "deploy": "sh bin/deploy.sh",
+    "deploy:dev": "sh bin/deploy.sh",
     "deploy:prod": "yarn lint && git add . && git commit -m \"chore(deploy): Merge staging -> prod\" &&git pull origin staging  && git checkout prod && git merge staging && git push origin prod && git checkout staging",
-    "dev": "yarn workspace @oasis/web dev",
+    "dev:web": "yarn workspace @oasis/web dev",
```

Comment:
Not needed, dev starts the dev server, just to keep it simple it can stay as dev.